src/main/java/easy/mind/com/api/Controller/UserController.java

```diff
@@ -0,0 +1,51 @@
+package easy.mind.com.api.Controller;
```

Comment:
Controller here starts with uppercase letter, every-every path should be lowercase